weaviate/collections/queries/fetch_object_by_id.py

```diff
@@ -286,6 +286,7 @@ def resolve_nested(obj: WeaviateField, prop: FromNested) -> dict:
         ret_refs = (
             parsed_refs if isinstance(parsed_refs, list) or parsed_refs is None else [parsed_refs]
         )
+        refs: Optional[dict] = None
```

Comment:
types for dict?

Reply:
Because it's an internal type, since the method is protected `_get_with_rest`, this type never makes it to the user; it would only benefit devs. Then, we'll be removing this code anyway soon so I didn't see a need for stronger types